src/Params/ADnoteParameters.cpp

```diff
@@ -81,7 +81,7 @@ static const Ports voicePorts = {
             if(obj->FmGn == NULL) return;
         data.obj = obj->FmGn;
         SNIP
-            OscilGen::realtime_ports.dispatch(msg, data);
+            OscilGen::ports.dispatch(msg, data);
```

Comment:
This dispatch should no longer occur given that the adnoteparameter ports are realtime and this would result in a realtime port invoking a non-realtime port.

Reply:
Solved (took only 3 weeks :p)

Reply:
Heh, faster than a lot of my recent contributions :p